% REQUIRED PACKAGES    -------------------------------------------------------------------------
% 
% 
% texlive-latex-base
% texlive-latex-recommended
% texlive-latex-extra
% texlive-fonts-recommended
% texlive-fonts-extra
% texmaker
% texmaker-data
% cm-super
% 
% 
% 
%%  FORMAT     ---------------------------------------------------------------------------------
\documentclass[11pt, a4paper, sans, blue]{moderncv}
\usepackage[english]{babel}	% language and dictionary setting
\usepackage[utf8]{inputenc}	% latin encoding
\usepackage[T1]{fontenc}	% T1 encoded CM-fonts, requires synaptic package cm-super
\usepackage{upgreek}		% upright standing small greek letters, usage with $ \upletter $
\usepackage[scale=0.78]{geometry} % Reduce document margins
\usepackage{hyphenat}
\usepackage{pdfpages}
\usepackage{ragged2e}
\moderncvstyle{casual} % CV theme - options include: 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{green}

%%  INFO       ---------------------------------------------------------------------------------
\firstname{Michael}
\familyname{Jahn}
\title{Curriculum vitae}
\address{Skogsbacken 20}{17241 Sundbyberg, Sweden}
\mobile{+46\,76\,7812578}
%\phone{}
\email{michael.jahn$@$scilifelab.se}
\photo[0.25\textwidth]{../figures/pic2_small.png}



\begin{document}

%%  OPTIONAL COVERLETTER    ---------------------------------------------------------------------
% include optional cover letter
%\input{coverletter_Muenster.tex}

\newpage

%%  ENTRIES     ---------------------------------------------------------------------------------

\maketitle

\setcounter{page}{1}


\section{Personal information}
\cvitem{Name}{Dr. rer. nat. Michael Jahn}
\cvitem{Date of birth}{Dec 31, 1985}
\cvitem{Born in}{Dresden, Germany}
\cvitem{Address}{Skogsbacken 20, 17241 Sundbyberg, Sweden}
\cvitem{Mobile phone}{+46\,76\,7812578}
\cvitem{Email}{michael.jahn$@$scilifelab.se}


\section{Current position}
\cvitem{}{\textbf{Post-doctoral fellow}, group of Prof.\,Paul Hudson, Science For Life Laboratory, Stockholm. Topic: Systems biology and metabolic engineering of photoautotrophic bacteria.}


\section{Education}
\cventry{July 10, 2015}{Graduated \textit{Doctor rerum naturalium}}{}{ at the University of Leipzig, Germany, with \textit{summa cum laude} (1.0)}{}{}
\cventry{2011--2015}{PhD studies}{}{Group of Prof.\,Susann Müller, Dept. Environmental Microbiology, Helmholtz-Centre for Environmental Research, Leipzig. \newline
	Thesis: 'Characterization of population heterogeneity in a model biotechnological process using \textit{Pseudomonas putida}'}{}{}
\cventry{2005--2011}{Diploma studies, biology}{}{Dresden University of Technology (TUD), main subjects: genetics, biochemistry, immunology. \newline
	Diploma thesis: 'Dynamic mating pheromone gradients for induction of mating projection and fluorescence in yeast', group of Prof.\,Gerhard Rödel, grade 1.2}{}{}
\cventry{1996--2004}{High school}{}{Romain-Rolland-Gymnasium, Dresden}{Abitur with grade 1.6}{}{}
\cventry{1992--1996}{Elementary school}{}{Dresden}{}{}


\section{Work experience}
\cventry{5/2016--now}{Post-doctoral researcher}{}{Systems biology analysis of cyanobacteria at the group of Prof.\,Paul Hudson, Science For Life Laboratory, Stockholm (belongs to KTH)}{}{}
\cventry{8--12/2009}{Erasmus internship}{}{Center of Excellence in Evolutionary Genetics and Physiology at the group of Prof.\,Mikko Nikinmaa, Dept. of Biology, University of Turku, Finland}{}{}
\cventry{2008--2009}{Student assistant}{}{Institute of Genetics}{group of Prof.\,Gerhard Rödel, Dept. of Biology, Dresden University of Technology (TUD)}{}
\cventry{2007--2008}{Student assistant}{}{Mitteldeutscher Praxisverbund Humangenetik, Dresden}{}{}
\cventry{2004--2005}{Voluntary service 'Freiwilliges Ökologisches Jahr'}{}{national park 'Sächsische Schweiz'}{tour guide and education}{}


\section{Leaves}
\cvitem{11/2019--3/2020}{Parental leave with 2\textsuperscript{nd} child Sophie}
\cvitem{9/2015--4/2016}{Parental leave with 1\textsuperscript{st} child Simon}


\section{Awards}
\cvitem{Nov 25, 2015}{\textbf{PhD-Award} 2015 by the Helmholtz-Centre for Environmental Research, Leipzig. Awarded annually for excellence of the dissertation, prize money 1,000 Euro.}


\section{Awarded grants}
\cvitem{Nov 11, 2019}{\textbf{Investigation of protein resource allocation in the model CO\textsubscript{2} fixing bacterium \textit{Ralstonia eutropha}} (Swedish: Undersökning av proteiners resursfördelning i en modell för CO\textsubscript{2}-fixerande bakterier, Ralstonia eutropha). \newline
Funding agency: FORMAS. Project number: 2019-01491. Project leader: Dr. Michael Jahn. Awarded funding for period 2020-01-01-2021-12-31. Total funds: 2,000,000 SEK (200,000 Euro).}


\section{Teaching and supervision}
\cvitem{Teaching}{Planned teaching activity for the \textbf{course CB2030 'Systems biology'} at Kungliga Tekniska Högskolan (KTH, together with Prof.\,Lukas Käll), study period winter 2019 -- spring 2020. Teaching activity was unfortunately postponed due to parental leave and COVID-19 outbreak. It is intended to resume teaching activity as soon as possible. Topics covered for this course among others: Pathway Analysis, Genome-scale metabolic models, Flux Balance Analysis.}

\cvitem{}{Introductory course on the programming language R and its application for the  analysis of flow cytometry data at the Helmholtz Centre for Environmental Research, Leipzig, 2015.}

\cvitem{}{Member of the PhD council and the Environmental Board of the Helmholtz Centre for Environmental Research, Leipzig, 2011-2015. Among other tasks providing guidance for starting PhD students.}

\cvitem{Supervision}{Supervised several master and intern students during my PhD and postdoctoral studies: \newline
\listitemsymbol Alexander Mattausch, intern biotechnology, University of Heidelberg, 2019\newline 
\listitemsymbol Timothy Bergmann, master student biotechnology (KTH, Stockholm), 2019\newline
\listitemsymbol Raquel Perruca, master student, Lund Technical University, 2018\newline
\listitemsymbol Carsten Vorpahl, master student University of Leipzig, 2015\newline\listitemsymbol Dominique Türkowsky, master student University of Leipzig, 2014\newline
}


\section{Scientific interests and expertise}

\cvitem{Sustainability}{Current overuse of (fossil) planetary resources and disposal of non-degradable waste into the environement makes the shift towards a sustainable economy mandatory. The concept of the \textbf{bioeconomy} entails that conventional chemical processes or materials be replaced with sustainable ones. Autotrophic microbes can play their roll as a versatile, efficient and biodegradable catalyst.}

\cvitem{Metabolic modeling, data science and visualization}
{High-throughput methods generate larger amounts of biological data than ever before. This poses problems, but also opportunities. I envision a close integration of experiments and data with \textbf{predictive mathematical models}, such as genome scale metabolic models. This allows to integrate the current state of knowledge into a coherent system, detect contradictions, and identify overarching themes in metabolism.}

\cvitem{Metabolic engineering}
{Microbial cells are 'programmed' for production of biomass, not fine chemicals. \textbf{Rational design and engineering of pathways} is necessary to turn photo-autotrophic bacteria into 'solar power plants'.}

\cvitem{Proteomics \& other 'omics}
{Proteomics, transcriptomics and metabolomics are probing the whole cell instead of isolated parts. This allows to discover biological relationships that were not considered in the original hypothesis, and gives an unbiased picture.}

\cvitem{Single cell variability}
{Previous work regarding \textbf{single cell variability} of \textit{S. cerevisiae}, \textit{E. coli}, and \textit{Pseudomonas putida} showed me that a population is never uniform but varies extensively in size, shape, productivity, and plasmid copy number. Future projects aiming to build improved cellular catalysts have to consider this variability.}

\cvitem{Reproducibility and Transparency}
{Adopting \textbf{open standards, sharing protocols and resources}, and depositing data and apps on publicly available sites has great advantages. It increases reach, trust of the scientific community, and reproducibility of results.}


\section{Qualifications and skills}
\cvitem{Courses}{Approved project manager for genetic works, according to German laws (\S ~14,15 GenTSV), 2015. \newline
Laser scanning microscopy, 1 week course, 2013. \newline
Scientific writing and project proposal writing, 2 week course , 2013.}
	
\cvitem{Model organisms}{Cultivation and molecular biology of cyanobacteria (e.g. \textit{Synechocystis sp.}, \textit{Synechococcus}), \textit{Cupriavidus necator}, \textit{Pseudomonas putida}, \textit{E.\,coli}}
	
\cvitem{Important techniques}{
Bioreactor cultivation, chemostat, turbidostat \newline
Mass spectrometry, proteomics \newline
Flow cytometry, cell sorting, automated imaging \newline
Data analysis and visualization \newline
Molecular biology work (DNA and protein extraction, cloning)
}

\cvitem{Programming lanuguages}{R (advanced), Python (moderate)}

\cvitem{Authored R packages}{
\href{https://github.com/m-jahn/SysbioTreemaps}{\color{darkgray}SysbioTreemaps} -- visualizing gene expression with treemaps\newline
\href{https://m-jahn.shinyapps.io/ShinyMC/}{\color{darkgray}ShinyMC} -- interactive app to monitor bioreactor cultivations\newline
\href{https://m-jahn.shinyapps.io/ShinyProt/}{\color{darkgray}ShinyProt} -- interactive app to visualize proteomics data\newline
\href{https://m-jahn.shinyapps.io/ShinyLib/}{\color{darkgray}ShinyLib} -- interactive app to visualize gene knockout library data\newline
}

\cvitem{Metabolic models}{
Improved \textit{Synechocystis} coarse grained (simplified) resource allocation model, for \href{https://github.com/m-jahn/cell-economy-model-legacy}{\color{darkgray}GAMS}, and \href{https://github.com/m-jahn/cell-economy-model}{\color{darkgray}ported to python}. \newline
%
Improved and updated \href{https://github.com/m-jahn/genome-scale-models}{\color{darkgray}genome scale model (GEM)} for \textit{Cupriavidus necator} a.k.a. \textit{Ralstonia eutropha}. \newline
%
Constructed new genome scale \href{https://github.com/m-jahn/genome-scale-models}{\color{darkgray}resource balance (RBA) model} for \textit{Cupriavidus necator} a.k.a. \textit{Ralstonia eutropha}
}


\section{Peer review and outreach}
\cvitem{Peer review}{Active reviewer for several journals including Microbial Cell Factories, Frontiers in Microbiology, Photosynthesis Research, and others. The full list of my review contributions is available via \href{https://publons.com/researcher/1192906/michael-jahn/}{\color{darkgray}Publons.org}}.

\cvitem{Platforms}{\href{https://m-jahn.github.io/}{\color{darkgray}My home page$\nearrow$}}
\cvitem{}{\href{https://github.com/m-jahn/}{\color{darkgray} My github page$\nearrow$}}
\cvitem{}{\href{https://orcid.org/0000-0002-3913-153X}{\color{darkgray}My ORCID page$\nearrow$}}
\cvitem{}{\href{https://publons.com/researcher/1192906/michael-jahn/}{\color{darkgray}My Publons page$\nearrow$}}
\cvitem{}{\href{https://www.researchgate.net/profile/Michael_Jahn}{\color{darkgray}My ResearchGate page$\nearrow$}}
\cvitem{}{\href{https://twitter.com/mich_jahn}{\color{darkgray}My Twitter page$\nearrow$}}


\section{Languages}
\cventry{German}{Native language}{}{}{}{}
\cventry{English}{Proficient user}{}{CEF level C1}{}{}
\cventry{Swedish}{Vantage user}{}{CEF level B2}{}{}{}

\newpage


% List of publications -----------------------------
\section{Publications}
%
% template:
%\cvitem{n}{\textbf{Jahn M}, Name N. \textit{title}. Journal, \textbf{year}.}

\cvitem{\listitemsymbol}{Yao L, Shabestary K, Björk SM, Asplund-Samuelsson J, Joensson HN, \textbf{Jahn M}, Hudson EP. \textit{Pooled CRISPRi screening of the cyanobacterium Synechocystis sp PCC 6803 for enhanced industrial phenotypes}. Nature Communications, \textbf{2019}.}

\cvitem{\listitemsymbol}{Karlsen J, Asplund-Samuelsson J, Thomas Q, \textbf{Jahn M}, Hudson EP. \textit{Ribosome Profiling of Synechocystis Reveals Altered Ribosome Allocation at Carbon Starvation}. MSystems 3, e00126-18, \textbf{2018}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Vialas V, Karlsen J, Maddalo G, Edfors F, Forsström B, Uhlén M, Käll L, Hudson EP. \textit{Growth of Cyanobacteria Is Constrained by the Abundance of Light and Carbon Assimilation Proteins}. Cell Reports 25, 478–486.e8., \textbf{2018}.}

\cvitem{\listitemsymbol}{Shabestary K, Anfelt J, Ljungqvist E, \textbf{Jahn M}, Yao L, Hudson EP. \textit{Targeted Repression of Essential Genes To Arrest Growth and Increase Carbon Partitioning and Biofuel Titers in Cyanobacteria}. ACS Synthetic Biology, 7, 1669–1675, \textbf{2018}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Vorpahl C, Hübschmann T, Harms H, Müller S. \textit{Copy number variability of expression plasmids determined by cell sorting and Droplet Digital PCR}.  Microbial Cell Factories, \textbf{2016}.}

\cvitem{\listitemsymbol}{Lindmeyer M, \textbf{Jahn M}, Vorpahl C, Müller S, Schmid A, Bühler B. \textit{Variability in subpopulation formation propagates into biocatalytic variability of engineered Pseudomonas putida strains}.  Frontiers in microbiology 6, \textbf{2015}.}

\cvitem{\listitemsymbol}{Lieder S, \textbf{Jahn M}, Koepff J, Müller S, Takors Ralf. \textit{Environmental stress speeds up DNA replication in Pseudomonas putida in chemostat cultivations}.  Biotechnology journal, \textbf{2015}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Günther S, Müller S. \textit{Non-random distribution of macromolecules as driving forces for phenotypic variation}. Current Opinion in Microbiology. 25:49-55, \textbf{2015}.}

\cvitem{\listitemsymbol}{Rödiger S, Burdukiewicz M, Blagodatskikh K, \textbf{Jahn M}, Schierack P. \textit{R as an environment for reproducible analysis of DNA amplification experiments}. R Journal 7/1:127-150, \textbf{2015}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Vorpahl C, Türkowsky D, Lindmeyer M, Bühler B, Harms H, Müller S. \textit{Accurate determination of plasmid copy number of flow-sorted cells using droplet digital PCR}. Analytical Chemistry 86:5969-76, \textbf{2014}.}

\cvitem{\listitemsymbol}{Lieder S, \textbf{Jahn M}, Seifert J, von Bergen M, Müller S, Takors R. \textit{Subpopulation-proteomics reveal growth rate, but not cell cycling, as a major impact on protein composition in} Pseudomonas putida \textit{KT2440}. AMB Express 4:71, \textbf{2014}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Seifert J, von Bergen M, Schmid A, Bühler B, Müller S. \textit{Subpopulation-proteomics in prokaryotic populations}. Current Opinion in Biotechnology 24:79-87, \textbf{2013}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Seifert J, Hübschmann T, von Bergen M, Harms H, Müller S. \textit{Comparison of preservation methods for bacterial cells in cytomics and proteomics}. Journal Of Integrated Omics 3:1-9, \textbf{2013}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Mölle A, Rödel G, Ostermann K. \textit{Temporal and spatial properties of a yeast multi-cellular amplification system based on signal molecule diffusion}. Sensors 13:14511-22, \textbf{2013}.}
\newpage

\section{Conference contributions}

\cvitem{May 22--24, 2019}{\textbf{NPC 14} - Nordic congress on photosynthesis, Turku, Finland. Short oral presentation.}

\cvitem{Oct 24--26, 2018}{\textbf{4th Applied Synthetic Biology meeting} - Toulouse, France. Oral presentation.}

\cvitem{Jul 20--22, 2015}{\textbf{Single Cell VI} -- 6\textsuperscript{th} International Conference on Analysis of Microbial Cells at the Single Cell Level, Retz (Austria). Oral presentation.}

\cvitem{Oct 15--17, 2014}{\textbf{DGFZ} -- 24\textsuperscript{th} Annual Conference of the German Society for Cytometry, Dresden. Oral presentation.}

\cvitem{Jul 13--16, 2014	}{\textbf{ECB16} -- 16\textsuperscript{th} European Congress on Biotechnology, Edinburgh. Oral presentation.}

\cvitem{May 26--28, 2014}{\textbf{DECHEMA} -- Biomaterials Made in Bioreactors, Dresden. Poster.}

\cvitem{Mar 06--08, 2013	}{\textbf{RPP7} -- 7\textsuperscript{th} European Conference on Recombinant Protein Production, Ulm. Oral presentation.}	

\cvitem{Jul 21--25, 2013	}{\textbf{FEMS} -- 5\textsuperscript{th} Congress of European Microbiologists, Leipzig. Poster.}

\cvitem{Oct 10--12, 2012}{\textbf{DGFZ} -- 22\textsuperscript{nd} Annual Conference of the German Society for Cytometry, Bonn. Oral presentation.}

\cvitem{Jun 23--27, 2012}{\textbf{CYTO 2012} -- 27\textsuperscript{th} Congress of the International Society for Advancement of Cytometry, Leipzig. Oral presentation.}


%\section{Contacts for Reference}
%\cvitem{1}{
%\textbf{Prof. Dr. Hauke Harms} \newline
%Department of Environmental Microbiology \newline
%Helmholtz Centre for Environmental Research -- UFZ \newline
%Permoserstraße 15 / 04318 Leipzig / Germany \newline
%Phone: +49\,341\,2351260 / Fax: +49\,341\,2351351 \newline
%Email: hauke.harms$@$ufz.de}

%\cvitem{2}{
%\textbf{Prof. Dr. Andreas Schmid} \newline
%Department Solar Materials \& MIKAT- Center for Biocatalysis \newline
%Helmholtz Centre for Environmental Research -- UFZ \newline
%Permoserstraße 15 / 04318 Leipzig / Germany \newline
%Phone: +49\,341\,2351246 / Fax: +49\,341\,235451286 \newline
%Email: andreas.schmid$@$ufz.de}


\end{document}
