% REQUIRED PACKAGES    -------------------------------------------------------------------------
% 
% 
% texlive-latex-base
% texlive-latex-recommended
% texlive-latex-extra
% texlive-fonts-recommended
% texmaker
% texmaker-data
% cm-super
% 
%
%
%%  FORMAT     ---------------------------------------------------------------------------------
\documentclass[11pt, a4paper, sans, blue]{moderncv}
\usepackage[english]{babel}	% language and dictionary setting
\usepackage[utf8]{inputenc}	% latin encoding
\usepackage[T1]{fontenc}	% T1 encoded CM-fonts, synaptic package cm-super!
\usepackage{upgreek}		% upright standing small greek letters, usage with $ \upletter $
\usepackage[scale=0.78]{geometry} % Reduce document margins
\usepackage{hyphenat}
\usepackage{pdfpages}
\usepackage{ragged2e}
\moderncvstyle{casual} % CV theme - options include: 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{green}

%%  INFO       ---------------------------------------------------------------------------------
\firstname{Michael}
\familyname{Jahn}
\title{Curriculum vitae}
\address{Skogsbacken 20}{17241 Sundbyberg, Sweden}
\mobile{+46\,76\,7812578}
%\phone{}
\email{michael.jahn$@$scilifelab.se}
\photo[0.25\textwidth]{../figures/pic2_small.png}


%%  ENTRIES     ---------------------------------------------------------------------------------
\begin{document}

%----------------------------------------------------------------------------------------
%	COVER LETTER
%----------------------------------------------------------------------------------------
\clearpage
\thispagestyle{empty}
\recipient{\color{color1} Name}{\color{color1} Institution \\ Street adress, City, Country} % Letter recipient
\date{\today} % Letter date
\opening{Dear recipient,} % Opening greeting
\closing{\\ With best regards,} % Closing phrase
\enclosure[Attached]{CV, list of publications and conference contributions, contacts for reference, PhD and diploma certificates, certificate 'Approved project manager for gene technology'.\\
} % List of enclosed documents

\makelettertitle % Print letter title
\justifying

I herewith want to apply for ...
%
%I have been working in the field of microbial biotechnology since I started studying Biology at Technical University Dresden, Germany. I obtained my PhD at the Helmholtz-Centre for Environmental Research, Leipzig, working in the frame of an EU project together with eight academic and corporate partners. This highly applied project focused on the establishment of the solvent-tolerant bacterium \textit{Pseudomonas putida} as a host organism for demanding bioprocesses. I applied flow cytometry, cell sorting, mass spectrometry and digital PCR to identify and characterize single cell variability. During this time I was able to co-author several publications in the field of microbial systems biology and single cell analysis. For the past two years I was working at Science For Science Life Lab, Stockholm, with the goal to characterize growth bottlenecks in a model cyanobacterium using a systems biology approach. 
% 
%During my research career I acquired experience with many important microbial strains regarding cultivation and molecular genetics, including the design of genetic constructs for \textit{E.\,coli}, \textit{P.\,putida} and \textit{S.\,cerevisiae}. I am experienced in the use of Omics technologies, especially mass spectrometry based proteomics, and have developed methods for combination of MS based proteomics with cell sorting. Furthermore, I have strong expertise in bioinformatic analysis of high-throughput data including the development of custom platforms for data processing and visualization using the programming language R. 
%
%My primary research interest is to use molecular biology for creating tailor-made microbes, and to employ such strains for the sustainable production of chemicals or biologics. The advertised position would therefore ideally complement my research background.
%I would be happy to discuss ideas regarding this project and look forward to hearing from you.
\makeletterclosing % Print letter signature

\newpage

\maketitle

\setcounter{page}{1}


\section{Personal information}
\cvitem{Name}{Michael Jahn, PhD}
\cvitem{Date of birth}{Dec 31, 1985}
\cvitem{Born in}{Dresden, Germany}
\cvitem{Current residence}{Skogsbacken 20, 17241 Sundbyberg, Sweden}
\cvitem{Mobile phone}{+46\,76\,7812578}
\cvitem{Email}{michael.jahn$@$scilifelab.se}


\section{Current position}
\cvitem{}{\textbf{Post-doctoral fellow}, Group of Prof Paul Hudson, Science For Life Laboratory, Stockholm. Topic: Systems biology and metabolic engineering of photoautotrophic bacteria.}


\section{Awards}
\cvitem{Nov 25, 2015}{\textbf{PhD-Award} 2015 by the Helmholtz-Centre for Environmental Research, Leipzig. Awarded annually for excellence of the dissertation.}


\section{Education}
\cventry{July 10, 2015}{Graduated \textit{Doctor rerum naturalium}}{}{ at the University of Leipzig, Germany, with \textit{summa cum laude} (1.0)}{}{}
\cventry{2011--2015}{PhD studies}{}{Group of Flow Cytometry (Prof. Susann Müller), Dept. Environmental Microbiology, 
	Helmholtz-Centre for Environmental Research, Leipzig. \newline
	Thesis: 'Characterization of population heterogeneity in a model biotechnological process using \textit{Pseudomonas putida}'}{}{}
\cventry{2005--2011}{Diploma studies, biology}{}{Dresden University of Technology (TUD), main subjects: genetics, biochemistry, immunology. 			\newline
	Diploma thesis: 'Dynamic mating pheromone gradients for induction of mating projection and fluorescence in yeast', group of Prof Gerhard Rödel, grade 1.2}{}{}
\cventry{1996--2004}{High school}{}{Romain-Rolland-Gymnasium, Dresden}{Abitur with grade 1.6}{}{}
\cventry{1992--1996}{Elementary school}{}{Dresden}{}{}


\section{Work experience}
\cventry{05/2016--now }{Post-Doctoral Research}{}{Systems biology analysis of cyanobacteria (Prof. P. Hudson), Science For Life Lab, Stockholm}{}{}
\cventry{08--12/2009}{Erasmus internship}{}{Center of Excellence in Evolutionary Genetics and Physiology (Prof. Nikinmaa), Dept. of Biology, University of Turku, Finland}{}{}
\cventry{2008--2009}{Student assistant}{}{Institute of Genetics}{Dept. of Biology at TUD}{}
\cventry{2007--2008}{Student assistant}{}{Mitteldeutscher Praxisverbund Humangenetik, Dresden}{}{}
\cventry{2004--2005}{Voluntary service 'Freiwilliges Ökologisches Jahr'}{}{national park 'Sächsische Schweiz'}{tour guide and education}{}


\section{Awarded grants}
\cvitem{11, 2019}{\textbf{Investigation of protein resource allocation in the model CO2 fixing bacterium Ralstonia eutropha} (Swedish: Undersökning av proteiners resursfördelning i en modell för CO2-fixerande bakterier, Ralstonia eutropha). Funding agency: FORMAS. Project number: 2019-01491. Project leader: Dr. Michael Jahn. Awarded funding for period 2020-01-01-2021-12-31: 2,000,000 SEK}


\section{Teaching and supervision}



\section{Qualifications and skills}
\cvitem{Courses}{Approved project manager for genetic works, according to German laws (\S ~14,15 GenTSV), 2015. \newline
	Laser scanning microscopy, 1 week intensive course, 2013. \newline
	Scientific writing, project proposal writing, 2013.}
\cvitem{Molecular genetics}{Handling of microbes (\textit{E.\,coli}, \textit{P.\,putida}, \textit{S.\,cerevisiae}) \newline
	Molecular genetics: DNA, RNA, protein extraction	and purification \newline
	Vector design, DNA cloning, recombinant gene expression \newline
	Droplet Digital PCR, qRT-PCR, gel electrophoresis}
\cvitem{High-throughput	 techniques}{Flow cytometry, cell sorting \newline
	Mass spectrometry, proteomics \newline
	Metabolic mapping, data mining, visualization \newline
	Automated microscopic imaging}
\cvitem{Software \& Programming}{Languages: R (advanced), Latex (advanced), Python (beginner) \newline
	Mass spectrometry: openMS, script-based tools
	Imaging: CellProfiler, ImageJ, Inkscape, GIMP \newline
	Flow cytometry: R flow packages, Summit, FlowJo \newline
	Genome and proteome databases: NCBI, DAVID, KEGG, BioCyc \newline}
\cvitem{Other}{Driver's license}


\section{Peer review and outreach}
\cvitem{Peer review}{I'm an active reviewer for several journals including Microbial Cell Factories, Frontiers, Photosynthesis Research, and others. The full list of my review contributions is available via \href{https://publons.com/researcher/1192906/michael-jahn/}{Publons.org}}. 

\cvitem{Platforms}{\href{https://github.com/m-jahn/}{\color{darkgray} My github page$\nearrow$}}
\cvitem{}{\href{https://orcid.org/0000-0002-3913-153X}{\color{darkgray}My ORCID page$\nearrow$}}
\cvitem{}{\href{https://publons.com/researcher/1192906/michael-jahn/}{\color{darkgray}My Publons page$\nearrow$}}
\cvitem{}{\href{https://www.researchgate.net/profile/Michael_Jahn}{\color{darkgray}My ResearchGate page$\nearrow$}}
\cvitem{}{\href{https://twitter.com/mich_jahn}{\color{darkgray}My Twitter account$\nearrow$}}


\section{Languages}
\cventry{German}{Native language}{}{}{}{}
\cventry{English}{Proficient user}{}{CEF level C1}{}{}
\cventry{Swedish}{Vantage user}{}{CEF level B2}{}{}{}


\newpage

% List of publications -----------------------------
\section{Publications}
%
% template:
%\cvitem{n}{\textbf{Jahn M}, Name N. \textit{title}. Journal, \textbf{year}.}

\cvitem{\listitemsymbol}{Yao L, Shabestary K, Björk SM, Asplund-Samuelsson J, Joensson HN, \textbf{Jahn M}, Hudson EP. \textit{Pooled CRISPRi screening of the cyanobacterium Synechocystis sp PCC 6803 for enhanced industrial phenotypes}. Nature Communications, \textbf{2019}.}

\cvitem{\listitemsymbol}{Karlsen J, Asplund-Samuelsson J, Thomas Q, \textbf{Jahn M}, Hudson EP. \textit{Ribosome Profiling of Synechocystis Reveals Altered Ribosome Allocation at Carbon Starvation}. MSystems 3, e00126-18, \textbf{2018}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Vialas V, Karlsen J, Maddalo G, Edfors F, Forsström B, Uhlén M, Käll L, Hudson EP. \textit{Growth of Cyanobacteria Is Constrained by the Abundance of Light and Carbon Assimilation Proteins}. Cell Reports 25, 478–486.e8., \textbf{2018}.}

\cvitem{\listitemsymbol}{Shabestary K, Anfelt J, Ljungqvist E, \textbf{Jahn M}, Yao L, Hudson EP. \textit{Targeted Repression of Essential Genes To Arrest Growth and Increase Carbon Partitioning and Biofuel Titers in Cyanobacteria}. ACS Synthetic Biology, 7, 1669–1675, \textbf{2018}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Vorpahl C, Hübschmann T, Harms H, Müller S. \textit{Copy number variability of expression plasmids determined by cell sorting and Droplet Digital PCR}.  Microbial Cell Factories, \textbf{2016}.}

\cvitem{\listitemsymbol}{Lindmeyer M, \textbf{Jahn M}, Vorpahl C, Müller S, Schmid A, Bühler B. \textit{Variability in subpopulation formation propagates into biocatalytic variability of engineered Pseudomonas putida strains}.  Frontiers in microbiology 6, \textbf{2015}.}

\cvitem{\listitemsymbol}{Lieder S, \textbf{Jahn M}, Koepff J, Müller S, Takors Ralf. \textit{Environmental stress speeds up DNA replication in Pseudomonas putida in chemostat cultivations}.  Biotechnology journal, \textbf{2015}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Günther S, Müller S. \textit{Non-random distribution of macromolecules as driving forces for phenotypic variation}. Current Opinion in Microbiology. 25:49-55, \textbf{2015}.}

\cvitem{\listitemsymbol}{Rödiger S, Burdukiewicz M, Blagodatskikh K, \textbf{Jahn M}, Schierack P. \textit{R as an environment for reproducible analysis of DNA amplification experiments}. R Journal 7/1:127-150, \textbf{2015}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Vorpahl C, Türkowsky D, Lindmeyer M, Bühler B, Harms H, Müller S. \textit{Accurate determination of plasmid copy number of flow-sorted cells using droplet digital PCR}. Analytical Chemistry 86:5969-76, \textbf{2014}.}

\cvitem{\listitemsymbol}{Lieder S, \textbf{Jahn M}, Seifert J, von Bergen M, Müller S, Takors R. \textit{Subpopulation-proteomics reveal growth rate, but not cell cycling, as a major impact on protein composition in} Pseudomonas putida \textit{KT2440}. AMB Express 4:71, \textbf{2014}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Seifert J, von Bergen M, Schmid A, Bühler B, Müller S. \textit{Subpopulation-proteomics in prokaryotic populations}. Current Opinion in Biotechnology 24:79-87, \textbf{2013}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Seifert J, Hübschmann T, von Bergen M, Harms H, Müller S. \textit{Comparison of preservation methods for bacterial cells in cytomics and proteomics}. Journal Of Integrated Omics 3:1-9, \textbf{2013}.}

\cvitem{\listitemsymbol}{\textbf{Jahn M}, Mölle A, Rödel G, Ostermann K. \textit{Temporal and spatial properties of a yeast multi-cellular amplification system based on signal molecule diffusion}. Sensors 13:14511-22, \textbf{2013}.}
\newpage

\section{Conference contributions}

\cvitem{May 22--24, 2019}{\textbf{NPC 14} - Nordic congress on photosynthesis, Turku, Finland. Short presentation.}

\cvitem{Oct 24--26, 2018}{\textbf{4th Applied Synthetic Biology meeting} - Toulouse, France. Presentation.}

\cvitem{Jul 20--22, 2015}{\textbf{Single Cell VI} -- 6\textsuperscript{th} International Conference on Analysis of Microbial Cells at the Single Cell Level, Retz (Austria), presentation.}

\cvitem{Oct 15--17, 2014}{\textbf{DGFZ} -- 24\textsuperscript{th} Annual Conference of the German Society for Cytometry, Dresden, presentation.}

\cvitem{Jul 13--16, 2014	}{\textbf{ECB16} -- 16\textsuperscript{th} European Congress on Biotechnology, Edinburgh,	 presentation.}

\cvitem{May 26--28, 2014}{\textbf{DECHEMA} -- Biomaterials Made in Bioreactors, Dresden, poster.}

\cvitem{Mar 06--08, 2013	}{\textbf{RPP7} -- 7\textsuperscript{th} European Conference on Recombinant Protein Production, Ulm, presentation.}	

\cvitem{Jul 21--25, 2013	}{\textbf{FEMS} -- 5\textsuperscript{th} Congress of European Microbiologists, Leipzig, poster.}

\cvitem{Oct 10--12, 2012}{\textbf{DGFZ} -- 22\textsuperscript{nd} Annual Conference of the German Society for Cytometry, Bonn, presentation.}

\cvitem{Jun 23--27, 2012}{\textbf{CYTO 2012} -- 27\textsuperscript{th} Congress of the International Society for Advancement of Cytometry, Leipzig, presentation.}


%\section{Contacts for Reference}
%\cvitem{1}{
%\textbf{Prof. Dr. Hauke Harms} \newline
%Department of Environmental Microbiology \newline
%Helmholtz Centre for Environmental Research -- UFZ \newline
%Permoserstraße 15 / 04318 Leipzig / Germany \newline
%Phone: +49\,341\,2351260 / Fax: +49\,341\,2351351 \newline
%Email: hauke.harms$@$ufz.de}

%\cvitem{2}{
%\textbf{Prof. Dr. Andreas Schmid} \newline
%Department Solar Materials \& MIKAT- Center for Biocatalysis \newline
%Helmholtz Centre for Environmental Research -- UFZ \newline
%Permoserstraße 15 / 04318 Leipzig / Germany \newline
%Phone: +49\,341\,2351246 / Fax: +49\,341\,235451286 \newline
%Email: andreas.schmid$@$ufz.de}


\end{document}
